src/assignments/6-array.js

```diff
@@ -21,3 +31,10 @@ min=1
 
 */
 // start here writing the code for program2
+
+const arr =[5,2,6,9];
+console.log(arr);
+const max = Math.max(...arr);
+const min = Math.min(...arr);
```

Comment:
write the function return min max number in the array. 
https://www.w3resource.com/c-programming-exercises/array/c-array-exercise-9.php
have a look at this and redo the task!